test/built-ins/Promise/prototype/then/rxn-handler-rejected-next.js

```diff
@@ -21,33 +21,33 @@ var promise = new Promise(function(_, reject) {
 var log = '';
 
 promise.then(function() {
-    log += 'A';
-  }, function() {
-    log += 'a';
-  });
+  log += 'A';
+}, function() {
+  log += 'a';
+});
 
 promise.then(function() {
-    log += 'B';
-  }, function() {
-    log += 'b';
-  });
+  log += 'B';
+}, function() {
+  log += 'b';
+});
 
 promise.then(function() {
-    log += 'C';
-  }, function() {
-    log += 'c';
-  });
+  log += 'C';
+}, function() {
+  log += 'c';
+});
 
 promise.then(function() {
-    $DONE('This promise should not be fulfilled.');
-  }, function() {
-    if (log !== 'abc') {
-      $DONE(
-        'Expected each "onFulfilled" handler to be invoked exactly once in series. ' +
-        'Expected: abc. Actual: ' + log
-      );
-      return;
-    }
+  $DONE('This promise should not be fulfilled.');
+}, function() {
+  if (log !== 'abc') {
+    $DONE(
+      'Expected each "onFulfilled" handler to be invoked exactly once in series. ' +
```

Comment:
This error message is incorrect; onFulfilled should be onRejected.

Reply:
another one to be fixed in a separate PR.